code/obj/item/Tumbling_Creeper_trap.dm

```diff
@@ -0,0 +1,334 @@
+/obj/item/Tumbling_Creeper
```

Comment:
Why isn't this a plant or something?

Reply:
Definitely no capital letters.

Reply:
> Why isn't this a plant or something?

As far as i know there is no generic plant subtype of `/obj/item`... although, to be honest, nothing is stopping me to add one, which i should probably do.

Reply:
there is a plant object, i just didn't saw it, i move this to that parent.

Reply:
Once #13617 got merged i can refactor these into `obj/item/plant` inna way it is not abhorrently ugly.